### PR TITLE
update outdated instructions

### PR DESCRIPTION
**Pull request by deric**
The README gives impression that this fork is also not maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,15 +39,15 @@ and drop it on your lxc templates path (usually `/usr/share/lxc/templates`)._
 ```sh
 git clone https://github.com/fgrehm/vagrant-lxc-base-boxes.git
 cd vagrant-lxc-base-boxes
-make precise
+make DEBIAN_BOXES=jessie debian
```
Comment (obnoxxx) [inline README.md]: This hunk seems unrelated to the actual commit of removing the 'unmaintained' notice from the README. Also I am not sure this change is appropriate.

File: README.md [hunk 2]
```diff
@@ -46,15 +39,15 @@ and drop it on your lxc templates path (usually `/usr/share/lxc/templates`)._
...
 ```
 
 By default no provisioning tools will be included but you can pick the ones
 you want by providing some environmental variables. For example:
 
 ```sh
 PUPPET=1 CHEF=1 SALT=1 BABUSHKA=1 \
-make precise
+make UBUNTU_BOXES=precise ubuntu
```
Comment (obnoxxx) [inline README.md]: This hunk seems unrelated to the actual commit of removing the 'unmaintained' notice from the README. Also I am not sure this change is appropriate.

**Comment by obnoxxx** [comment]
Thanks for the patch!
(And apoplogies for the late reply..)

See inline comment: the first hunk of the patch is good, but the other two line-changes do not belong there, I think. Would you mind sending an updated pull request with only the first hunk?

Thanks!

**Comment by deric** [comment]
Sorry, it's quite a while ago. I'm pretty sure I've run into problems with those examples. But I don't recall what is was. I've updated the commit.

**Comment by obnoxxx** [comment]
Merged, thanks!
